service/src/test/java/bio/terra/cbas/initialization/TestCloneRecoveryTransactionService.java

```diff
@@ -46,6 +46,8 @@ void convertToTemplate() {
     verify(runDao).deleteRun(run1.runId());
     verify(runDao).deleteRun(run2.runId());
     verify(runDao).deleteRun(run3.runId());
+    verify(methodDao).unsetLastRunSetId(method.methodId());
```

Comment:
I would add `@ExtendWith(MockitoExtension.class)` to your class definition, that makes sure everything you set up is called, and nothing you don't set up is called, without having to explicitly call like this to check

Reply:
just talked about this offline with Chris, but for the record: I'm adding this extension, but *not* removing any of these calls to `verify`, because they are on void methods, and it's not clear how the mocks would be set up to specify the particular inputs I'm expecting for those methods